src/App.tsx

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
-
 import './App.scss';
+import { Post } from './types/Post';
+
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostList } from './components/PostList';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+const preparedPosts: Post[] = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  comments: comments.filter(comment => comment.postId === post.id),
```

Comment:
"comments: comments" Same name of variables in one string. Maybe for a better code view, you should use different names.

Reply:
Here you create an array of comments that belongs to a specific post and later don`t use it.